src/main/java/ch/njol/skript/events/EvtMove.java

```diff
@@ -42,47 +41,92 @@ public class EvtMove extends SkriptEvent {
 			events = CollectionUtils.array(PlayerMoveEvent.class, EntityMoveEvent.class);
 		else
 			events = CollectionUtils.array(PlayerMoveEvent.class);
-
-		Skript.registerEvent("Move", EvtMove.class, events, "%entitydata% (move|walk|step)")
-			.description("Called when a player or entity moves.",
-				"NOTE: Move event will only be called when the entity/player moves position, not orientation (ie: looking around).",
-				"NOTE: These events can be performance heavy as they are called quite often.",
-				"If you use these events, and later remove them, a server restart is recommended to clear registered events from Skript.")
-			.examples("on player move:",
-				"\tif player does not have permission \"player.can.move\":",
-				"\t\tcancel event",
-				"on skeleton move:",
-				"\tif event-entity is not in world \"world\":",
-				"\t\tkill event-entity")
-			.requiredPlugins("Paper 1.16.5+ (entity move)")
-			.since("2.6");
+		Skript.registerEvent("Move / Rotate", EvtMove.class, events,
+				"%entitydata% (move|walk|step|rotate:(look[ing] around|rotate))",
```

Comment:
I don't know if `on player look around` sounds quite right, maybe something like turn?
I also don't want this to get confused with the actual "look at" behaviour event, where an animal will look directly at the player.

Reply:
If you can't think of anything better don't worry about it.

Reply:
`turn` sounds like a better fit.